components/button/style/mixin.less

```diff
@@ -24,12 +24,12 @@
   .button-color(@color; @background; @background);
   &:hover,
   &:focus {
-    .button-color(@color; ~`colorPalette("@{background}", 5)`; ~`colorPalette("@{background}", 5)`);
+    .button-color(@color; @primary-5; @primary-5);
```

Comment:
I think you make a mistake, you can just customize theme with @background, and these code means that we want these color follow @background